src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorRequestsServiceImpl.java

```diff
@@ -91,18 +91,11 @@ private void processEachPaymentWorksNewVendorRequestIntoKFS(List<String> identif
                 }
                 if (pmwNewVendorSaveToStagingTableWasSuccessful(savedStgNewVendorRequestDetailToProcess, reportData)) {
                     if (pmwNewVendorRequestProcessingIntoKfsWasSuccessful(savedStgNewVendorRequestDetailToProcess, reportData)) {
-                        getPaymentWorksWebServiceCallsService().sendProcessedStatusToPaymentWorksForNewVendor(savedStgNewVendorRequestDetailToProcess.getPmwVendorRequestId());
-                    } else {
-                        getPaymentWorksWebServiceCallsService().sendRejectedStatusToPaymentWorksForNewVendor(savedStgNewVendorRequestDetailToProcess.getPmwVendorRequestId());
+                        LOG.info("processEachPaymentWorksNewVendorRequestIntoKFS, successfully processed vendor for pmwNewVendorRequestId: " + pmwNewVendorRequestId);
                     }
-                } else {
-                    // save of pmw data to staging table failed for some reason, cannot process or track request
-                    getPaymentWorksWebServiceCallsService().sendRejectedStatusToPaymentWorksForNewVendor(savedStgNewVendorRequestDetailToProcess.getPmwVendorRequestId());
                 }
-            } else {
-                // either duplicate request or data issue exists preventing save to staging table
-                getPaymentWorksWebServiceCallsService().sendRejectedStatusToPaymentWorksForNewVendor(stgNewVendorRequestDetailToProcess.getPmwVendorRequestId());
             }
```

Comment:
If the log file output you saw when validating locally did not show which PMW vendor ID encountered the two "else" conditions above, I would suggest adding Log.info messages in both those "else" condition locations. I realize that it may look like noise but it will definitively show which processing path each PMW vendor took in the batch job with this new logic.

Reply:
Ok, I'll look into adding in the extra logging.